installer/conf/omsagent.conf

```diff
@@ -84,9 +84,11 @@
   buffer_type file
   buffer_path /var/opt/microsoft/omsagent/state/out_oms_blob*.buffer
   buffer_queue_limit 10
+  buffer_queue_full_action drop_oldest_chunk
   flush_interval 20s
   retry_limit 10
   retry_wait 30s
+  max_retry_wait 9m
```

Comment:
I tried fixing the rebase issue but with the remote branch already reflected the change, I could not successfully change and submit a new PR. I am stil looking for a way to fix the mistake I made or git fetch and not rebase.